test/_init_tests.js

```diff
@@ -106,6 +106,18 @@ class TestConfig {
       return fn.apply(this, arguments);
     });
   }
+
+  /** @private */
+  isAgentMatched_(agent) {
```

Comment:
oh didn't know about `platform.js`. Yeah, should be able too. Wait for the next patch before reviewing..